package.json

```diff
@@ -40,7 +40,9 @@
   },
   "repository": "/generator-terra-module",
   "scripts": {
-    "prepublish": "gulp prepublish",
+    "release:major": "npm test && gulp prepublish && npm version major -m \"Released version %s\" && npm publish && git push --follow-tags",
```

Comment:
It looks like they should.
<img width="747" alt="screen shot 2017-05-25 at 3 40 29 pm" src="https://cloud.githubusercontent.com/assets/633148/26469696/92aa6ba4-4160-11e7-87f6-1ee169428de1.png">